app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -295,11 +295,20 @@ <h2>Upload Sessions</h2>
                  aria-labelledby="v-pills-questions-tab">
                 <h2>Questions</h2>
 
-                <ul class="list-group list-group-flush">
+                <script src="https://cdn.jsdelivr.net/npm/sortablejs@latest/Sortable.min.js"></script>
```

Comment:
Messily added this in-line. This is of course not acceptable? What is the way to do this? I saw something like this:

~~~~~.py
class Media:
    JS = ("something.js", )
    CSS = ("else.css",)
~~~~~

Is that the answer?

Reply:
Yes, you're hand-rolling a form here, it's better to let django render it. Create the form in the view and add it to the context.

Reply:
Also, it's best to serve it ourselves, these free CDNs are not reliable.

Reply:
You can add it to packages.json.